.github/workflows/windows.yml

```diff
@@ -22,6 +22,10 @@ jobs:
       fail-fast: false
       matrix:
         combo:
+          - otp-version: 'latest'
+            elixir-version: 'latest'
+            rebar3-version: 'latest'
+            os: 'windows-latest'
```

Comment:
⚠️ this change has to be reversed, as we won't be doing "online" integration tests, at this moment. Preferably, but it's on you to decide about your own time and energy, we'd have them in the test suites, using the mock `builds.txt`.

Reply:
That has already been tested; what this was catching was getOTPVersions that might be broken. But as I said in my prior PR, I'm not looking to do any major refactoring here.